src/components/collection_voting_slab/ResearchProposalVoteSlab.js

```diff
@@ -1,10 +1,18 @@
-import {useState} from "react";
+import {useEffect, useState} from "react";
 import {a, div, h, span} from "react-hyperscript-helpers";
 import {DataUseTranslation} from "../../libs/dataUseTranslation";
-import ld, {isEmpty, isNil} from "lodash";
+import {isEmpty, isNil, flatMap, map, keys} from "lodash";
```

Comment:
As a general preference, I like the fp versions of these - they're easier to read then the plain nested lodash versions.